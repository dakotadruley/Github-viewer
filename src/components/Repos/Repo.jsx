import React from 'react';
import PropTypes from 'prop-types';

const Repo = ({ name, url }) => (
  <a href={url}>{name}</a>
);

Repo.propTypes = {
  name: PropTypes.string.isRequired,
  url: PropTypes.string.isRequired,
};

export default Repo;
