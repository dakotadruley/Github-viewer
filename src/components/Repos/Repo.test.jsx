import React from 'react';
import { shallow } from 'enzyme';
import Repo from './Repo';

describe('Repo', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(<Repo
      name="dakota-app"
      url="url" />);
    expect(wrapper).toMatchSnapshot();
  });
});
