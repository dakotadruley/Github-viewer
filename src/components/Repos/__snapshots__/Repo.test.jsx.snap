// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Repo matches a snapshot 1`] = `
<a>
  dakota-app
</a>
`;
